src/test/kotlin/dev/supergrecko/kllvm/ir/values/GlobalVariableTest.kt

```diff
@@ -28,9 +32,10 @@ class GlobalVariableTest {
             assertFalse { isThreadLocal() }
             assertFalse { isExternallyInitialized() }
 
-            val initializer = value.getInitializer()
-                ?.asIntValue()
-                ?.getSignedValue()
+            val initializer = when (val init = value.getInitializer()) {
+                is Some<Value> -> init.t.asIntValue().getSignedValue()
```

Comment:
The thing I kind of dislike is the .t on the ArrowKt Option<T> type. This can definitely be worked around if we run our own Option<T>